data-es6.js

```diff
@@ -13910,6 +13931,7 @@ exports.tests = [
       */},
       res: {
         xs6:         false,
+        chrome50:    true,
```

Comment:
Thanks @zloirock, I'll look into it. I know Chrome 48 threw some exceptions that broke some sites, and Chrome 49 will revert this in a way that the spec does not describe, but I did not know of a change here.